keps/2724-topology-aware-scheduling/README.md

```diff
@@ -762,6 +929,41 @@ We recommend the BestFit algorithm for most of use cases, however we give more f
 Based on the collected feedback we will introduce TAS configuration that would allow user to select the desired algorithm.
 Eventually we'll remove the feature as they will be no longer need when we implement API for TAS configuration.
 
+### Two-level scheduling
+In consideration of a [Story 5](#story-5) a two-level scheduling is introduced. 
+We introduce a notion of PodSet Slice, which is a subset of PodSet. All slices
+have the same size and evenly divide the PodSet. To simplify implementation we 
+are allowing only for a "required" topology for slice. The requested topology 
+level for slices has to be on the same or below level of the main topology.
+
+Requesting slice topology changes the way algorithm chooses domains. In all 
+modes in [Computing the assignment](#computing-the-assignment) algorithm greedily
+selects domains based on the free capacity (either most or least). However, with
+slices it considers the amount of slices that can fit into a domain as free 
+capacity, but if that number is equal then it prioritizes domains with fewer 
+remaining resources.
+
+#### Example
+Consider a rack with nodes that can accommodate 6, 5, 4, 3, and 2 pods respectively. 
+The required topology for slice is "host" and the podset slice size is 2.
+
+As an example let's analyze the assignment based on the size of a podset and selected 
+mode. See the table below where the numbers are pods assigned to a particular node.
+The header for columns dedicated to nodes correspond node's initial capacity.
+
+| mode                   | PodSet size | 6 | 5 | 4 | 3 | 2 |
```

Comment:
What does `PodSet size` mean? PodSet count? PodSet Slice size?

Reply:
> I'm a new contributor, so I might be mistaken, but looking here https://github.com/kubernetes-sigs/kueue/blob/main/pkg/controller/jobs/jobset/jobset_controller.go#L232, we multiply parallelism/completions and replicas

yes, that's right. The above yaml place was not correct. Let me show those more accurately.
I wanted to show in the following:

- PodSet Count: `.spec.podSet[*].count` (Workload)
- Pod Count: `.spec.replicatedJobs[*].replicas` (JobSet) × `.spec.replicatedJobs[*].template.spec.parallelism`
- Pod Count more generic: `.spec.podSet[*].count` (Workload) × `.spec.replicatedJobs[*].template.spec.parallelism`